### ISSUE TITLE
GLEW and QWT:    glew32.dll and qwt.dll  not working on Windows 10

### ISSUE THREAD

**Issue by ig-or** [JuliaPackaging/Yggdrasil#3880]
Those library cannot be used because looks like they are installed with not correct permissions. "read only" for everyone and "read and write" for me. All the other DLL files get "Read and execute" permissions somehow.

I noticed how glew is created:  `LibraryProduct(["libGLEW", "glew32"], :libGLEW),`
and    qwt:    ` LibraryProduct(["libqwt", "qwt"], :qwt),`
and marble library(which works good): ` LibraryProduct("libmarblewidget-qt5", :marblewidget),`
Maybe this difference could be the reason?

Best regards.

**Comment by giordano** [comment]
> Those library cannot be used because looks like they are installed with not correct permissions. "read only" for everyone and "read and write" for me. All the other DLL files get "Read and execute" permissions somehow.

Those packages have been built before https://github.com/JuliaPackaging/BinaryBuilder.jl/pull/992

>  Maybe this difference could be the reason?

No, there is no functional difference, they're just different constructors of the same data type

**Comment by ig-or** [comment]
So what could be the correct (or fast and ugly) way to try to resole this? 

QWT script is  using qmake, maybe this is the issue?
```
script = raw"""
cd $WORKSPACE/srcdir/qwt
qmake
make -j${nproc}
make install
"""
```

For GLEW, its make file have "dist-win32" as a target, but this is not used in a jl build script..    maybe this is the issue?
Maybe we can just make "chmod +x *.dll"   as was suggested in https://github.com/JuliaPackaging/Yggdrasil/pull/2445 ?

**Comment by giordano** [comment]
Fixed by #3886
